Tests/GeoLib/TestAABB.cpp

```diff
@@ -220,3 +238,47 @@ TEST(GeoLib, AABBAllPointsWithNegativeCoordinatesII)
 	ASSERT_NEAR(-1.0, max_pnt[2], std::numeric_limits<double>::epsilon());
 }
 
+TEST(GeoLib, AABBSinglePoint)
+{
```

Comment:
This test creates an AABB containing a single point.
This point is moved in every space direction to the next possible double value.
It is checked that the AABB contains the point iff it has not been moved.